react/index.js

```diff
@@ -286,7 +286,31 @@ class ProductSummary extends Component {
       'bb b--muted-4 ma2': showBorders
     })
 
-    const style = { visibility: showBuyButton ? 'visible' : 'hidden' }
+    const buyButtonStyle = { visibility: showBuyButton ? 'visible' : 'hidden' }
```

Comment:
Do not you think that should be better to use tachyons css to hide the component instead of styled fields?